internal/adapters/runtime/kubernetes/scheduler.go

```diff
@@ -63,3 +170,53 @@ func (k *kubernetes) DeleteScheduler(ctx context.Context, scheduler *entities.Sc
 
 	return nil
 }
+
+func (k *kubernetes) MitigateDisruption(
+	ctx context.Context,
+	scheduler *entities.Scheduler,
+	roomAmount int,
+	safetyPercentage float64,
+) error {
+	if scheduler == nil {
+		return errors.NewErrInvalidArgument("empty pointer received for scheduler, can not mitigate disruptions")
+	}
+
+	incSafetyPercentage := 1.0
+	if safetyPercentage < 0.0 {
+		k.logger.Warn("invalid safety percentage, default to 0 without safety", zap.Float64("safetyPercentage", safetyPercentage))
+		safetyPercentage = 0.0
+	}
+	incSafetyPercentage += safetyPercentage / 100
```

Comment:
By this, if no value is given through the configuration, we won't have any buffer, is that correct? If so, why we decided to not have any buffer?

Reply:
I've added a default safety to 5% in case the value is `0.0` (default if not set by the user) or below minimum